xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/XmlNodesSelectorTest.cs

```diff
@@ -0,0 +1,100 @@
+using SelectingXmlNodesWithXpath;
+using System.Xml;
```

Comment:
These should move to the global usings file, since we have one and these will apply across the board to anything within our unit tests